sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/VolumeEvaluationStage.java

```diff
@@ -4,6 +4,7 @@
 import com.mesosphere.sdk.offer.*;
 import com.mesosphere.sdk.specification.VolumeSpec;
 import org.apache.mesos.Protos.Resource;
+import org.apache.mesos.Protos.ResourceProviderID;
```

Comment:
Now that this is using multiple `Protos` types, can just import `Protos` here and then refer to the types as `Protos.X` below.